astroplan/plots/sky.py

```diff
@@ -195,7 +195,7 @@ def plot_sky(target, observer, time, ax=None, style_kwargs=None,
     ]
 
     theta_labels = []
-    for chunk in range(0, 7):
+    for chunk in range(0, 8):
```

Comment:
@bmorris3 @wtgee -- made this change to match the tick labels that are defined below. Without it, the doc build fails.